packages/twenty-chrome-extension/src/options/Sidepanel.tsx

```diff
@@ -46,44 +46,101 @@ const Sidepanel = () => {
   const iframeRef = useRef<HTMLIFrameElement>(null);
```

Comment:
In the app how do I access the option panel to change the server's url? I couldn't find it

Reply:
Added the settings page where we change urls, along with the button to navigate to the said page